docs/release-notes/gateway.rst

```diff
@@ -0,0 +1,13 @@
+:orphan:
+
+**New Features**
+
+-  Kubernetes: Introduced the Internal Task Gateway feature which allows Determined tasks running on
+   remote K8s clusters to be exposed to the Determined master and proxies. This feature supports
+   multi-resource manager setups by configuring a Gateway controller in the external K8s cluster.
+
```

Comment:
:orphan:

**New Features**

-  Kubernetes: The Internal Task Gateway feature enables Determined tasks running on remote Kubernetes clusters to be exposed to the Determined master and proxies. This feature facilitates multi-resource manager setups by configuring a Gateway controller in the external Kubernetes cluster.